src/_includes/content/boiler.md

```diff
@@ -0,0 +1,24 @@
+## Introduction
+
+B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.
+
+B01LER provides pre-built components that exist in design in the form of a [component library in Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig)), as well as in [code in a repository](https://github.com/deven-org/B01LER-Kitchen), which is also [displayed in the web using Storybook](https://b01ler.onrender.com/). Both the components in design and in code are connected via a token system.
+
+A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
```

Comment:
That first sentence got quite long with lots of stuff happening, which makes it hard to read to me. I would reduce the number of links to the main two (not the internal one), to not distract too much with extra information here.

I'm also not certain if "in design" and "in code" work as a key phrase here.

How about this:

```suggestion
B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/B01LER-Kitchen).

The design and code implementations of the components are connected via a shared **token system**. A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
```

Rendered:

---

B01LER provides pre-built components that exist **as designs** in the form of a [component library in Figma](https://boilerds.com/B01LER%20%20%5Bv0.1%20-%20alpha%20release%5D.fig), and **as code** in the form of a [forkable repository](https://github.com/deven-org/B01LER-Kitchen).

The design and code implementations of the components are connected via a shared **token system**. A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.

Reply:
You are right, thanks for the suggestion, will go with that